homeassistant/components/todoist/strings.json

```
@@ -84,6 +96,34 @@
         "reminder_date": {
           "name": "Reminder date",
           "description": "When the user should be reminded of this task, in format YYYY-MM-DDTHH:MM:SS, in UTC timezone."
+        },
+        "reminder_zone": {
+          "name": "Reminder zone",
+          "description": "The zone that should trigger the reminder. Note that location data will be shared with Todoist."
+        },
+        "reminder_location_name": {
+          "name": "Reminder location name",
+          "description": "An alias name for the location that should trigger the reminder."
+        },
+        "reminder_latitude": {
+          "name": "Reminder latitude",
+          "description": "The latitude of the location that should trigger the reminder. Note that location data will be shared with Todoist."
+        },
+        "reminder_longitude": {
+          "name": "Reminder longitude",
+          "description": "The longitude of the location that should trigger the reminder. Note that location data will be shared with Todoist."
+        },
+        "reminder_radius": {
+          "name": "Reminder radius",
+          "description": "The radius of the location that should trigger the reminder."
+        },
+        "reminder_location_direction": {
+          "name": "Reminder location direction",
+          "description": "Should the reminder be triggered when entering or leaving the zone?"
```

To match the usual style in Home Assistant this should be reworded:

```suggestion
          "description": "Whether the reminder should be triggered when entering or leaving the zone."
```